Sources/SwiftGen/Commander/CustomCommands.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CustomCommands.swift
+//  swiftgen
+//
+//  Created by Philippe Bernery on 15/01/2018.
+//  Copyright © 2018 Backelite. All rights reserved.
+//
+
+import Commander
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+
+private let templatePathOption = Option<String>(
+  "templatePath", default: "", flag: "p",
+  description: "The path of the template to use for code generation."
+)
+
+private let paramsOption = VariadicOption<String>(
+  "param", default: [],
+  description: "List of template parameters"
+)
+
+let customCommand = Commander.command(
```

Comment:
Maybe try to use the existing framework for parser commands? I know you don't have a resource "path" parameter, but maybe create a more generic protocol (or struct) for commands with no resources. @AliSoftware?

Reply:
Never thought we could have subcommands not taking any path, but yeah would be nicer to have all subcommand code build on the same type of similar code.

Reply:
Ok. I'll change it depending on how this PR may be accepted or not. See discussion follow-up below.